server.py

```diff
@@ -21,6 +24,18 @@ def download(number):
 @app.route('/upload', method = 'POST')
 def file_upload():
     uploaded = request.files.get('upload').file.read() #uploaded outputs by user
+    u_name = request.forms.get('username') #accepting username
+
+    # storing usernames' submission along with the timestamp at which they submit it
+    global usernames
+    time = datetime.datetime.now()
+    t = time.strftime("%x") + ' ' + time.strftime("%I") + ":" + time.strftime("%M") + ":" + \
```

Comment:
you can just store the time object no?

Reply:
`2019-01-23 23:35:31.237667`

This is a example of what `'time' stores actually. The only issue I am having in this is the over precision of the seconds. That is what I wanted to get rid of. I can store the time object if it is okay to have such precision.

Reply:
We can store the object as it is since right now we are not using it for anything. Later on we can decide to convert to string depending on what we use it for.